subprojects/build-cache/src/integTest/groovy/org/gradle/caching/internal/BuildCacheBuildOperationsIntegrationTest.groovy

```diff
@@ -296,6 +306,9 @@ class BuildCacheBuildOperationsIntegrationTest extends AbstractIntegrationSpec {
         localStore << [
             true, false, false, false
         ]
+        expectDeprecation << [
```

Comment:
Can we use a `where` clause with columns? Matching up the different lists doesn't seem very nice.

Reply:
Sure can. The idea was that this will come out as soon as we remove custom local build caches, but I see that even then using columns might be nicer, so no objection.